packages/expo/src/ImagePicker/ExponentImagePicker.web.ts

```diff
@@ -1,5 +1,83 @@
+import uuidv4 from 'uuid/v4';
+
+import { CameraOptions, ImageLibraryOptions, ImageResult } from './ImagePicker.types';
+import MediaTypeOptions from './MediaTypeOptions';
+
+const MediaTypeInput = {
+  [MediaTypeOptions.All]: 'video/*,image/*',
+  [MediaTypeOptions.Images]: 'image/*',
+  [MediaTypeOptions.Videos]: 'video/*',
+};
+
 export default {
   get name(): string {
     return 'ExponentImagePicker';
   },
+  async launchImageLibraryAsync({
+    mediaTypes,
```

Comment:
```suggestion
    mediaTypes = MediaTypeOptions.All,
```
to make it more visible?